dashboard/config/generate-config-kmsan-from-kasan.sh

```diff
@@ -22,6 +22,12 @@ scripts/kconfig/merge_config.sh -m .config ${KMSAN_ADD}
 make ${MAKE_VARS} olddefconfig
 
 util_add_usb_bits
+
+# KMSAN doesn't support ORC unwinder (https://github.com/google/kmsan/issues/48).
+scripts/config -d CONFIG_UNWINDER_ORC
+scripts/config -e CONFIG_UNWINDER_FRAME_POINTER
+make ${MAKE_VARS} olddefconfig
```

Comment:
This should be in `bits-kmsan.config`. We have all such individual configs there. Also will save on 1 olddefconfig run.

Reply:
Done.